### ISSUE TITLE
Not compatible with Templater v1.6.0

### ISSUE THREAD

**Issue by MartynKeigher** [Vinzent03/obsidian-hotkeys-for-templates#5]
Hey all, 

Updated to **[Templater v1.6.0](https://github.com/SilentVoid13/Templater)** earlier today and as a result the following feature, in this plug, did not work...

![image](https://user-images.githubusercontent.com/3298134/116929787-e9717f00-ac2c-11eb-8988-e663fe6f6615.png)

I have keyboard shortcuts tied to these too and after updating Tempalter … all I got was an Untitled page in the folder specified. 

So this was the code I adjusted to fix it...

In `.obsidian\plugins\obsidian-hotkeys-for-templates\main.js`, I adjusted LINE **2372** from,
```
case 1: return [4 /*yield*/, this.templaterPlugin.parser.replace_templates_and_append(file)];
```
to:
```
case 1: return [4 /*yield*/, this.templaterPlugin.templater.append_template(file)];
```

Regards, 
://mk

**Comment by Vinzent03** [comment]
Thanks for the hind. Will fix it now.

**Comment by Vinzent03** [comment]
Fixed in [1.2.1](https://github.com/Vinzent03/obsidian-hotkeys-for-templates/releases/tag/1.2.1)
